lib/game.rb

```diff
@@ -14,6 +15,7 @@ def initialize
     @finish = false
     @current_player = @player_1
     @validator = Validator.new(@board)
+    @rules = Rules.new(@board)
   end
 
   def launch_game
```

Comment:
Something to consider for the future: 
@breadoliveoilsalt @twopoint718 
This method should have a test so this would be good refactoring/moving logic to own place practice. Testing this might involve a lot of mocking to get desired results so this would be good practice for that as well.

Reply:
feature needs to be developed in a future story is time permits